src/go/rpk/pkg/config/config.go

```diff
@@ -143,6 +143,21 @@ func AvailableModes() []string {
 	}
 }
 
+// FileOrDefaults return the configuration as read from the file or
+// the default configuration if there is no file loaded.
+func (c *Config) FileOrDefaults() *Config {
+	if c.File() != nil {
+		cfg := c.File()
+		cfg.loadedPath = c.loadedPath
```

Comment:
Can this and the following line be added after line 399 in params.go?

Reply:
Added only

`	c.file.loadedPath = abs
`

in params.go

Reply:
Going to handle this in https://github.com/redpanda-data/redpanda/issues/5458